aidants_connect_web/templates/aidants_connect_web/mandat/mandat.html

```diff
@@ -23,30 +25,54 @@ <h2 class="subtitle">Mentions à lire à la personne accompagnée</h2>
     </div>
   </div>
 </section>
-<section id="select_procedure" class="section section-grey">
-  <div class="container container-small">
+<section id="mandat_specifications" class="section section-grey">
+  <div class="container">
     <form method="post">
       {% if notification %}
-        <div class="notification">{{ notification }}</div>
+      <div class="notification">{{ notification }}</div>
       {% endif %}
-      <h2>Choisir la démarche</h2>
-      {% if form.errors %}
+      <div id="demarches" class="tiles">
+        <h2>Étape 1: Sélectionnez la ou les démarche(s)</h2>
+        {% if form.errors %}
         <div class="notification">
-          {{ form.errors }}
+        {{ form.errors }}
+        </div>
+        {% endif %}
+        {% csrf_token %}
+        <div id="demarches_list" class="grid">
+            {% for value, label in form.perimeter.field.choices %}
+            <div id="{{ value }}" class="tile">
+                <input id="button-{{ value }}" type="checkbox" value="{{ value }}" name="perimeter"/>
+                <label id="label-{{ value }}" for="button-{{ value }}">
+                    <div style="display: flex; flex-direction: column">
+                        <div style="display: flex; align-items: center">
+                            <img src={{ label.icon }}>
+                            <p>{{ label.titre }}</p>
+                        </div>
+                        <div>
+                            {{ label.description }}
+                        </div>
+                   </div>
+                </label>
+            </div>
+            {% endfor %}
+        </div>
+        </div>
+      <div id="duration" >
+            <h2 style="margin-top: 2em">Étape 2: Choisissez la durée du mandat</h2>
+            <div id="duration_list" class="grid" >
+              {% for value, label in form.duration.field.choices %}
+                  <div id="{{ value }}" class="tile" style="display: flex; flex-direction: row">
+                    <input id="button-{{ value }}" type="radio" value="{{ value }}" name="duration"/>
+                    <label id="label-{{ value }}" for="button-{{ value }}"><b>{{ label.title }}</b> {{ label.description }}</label>
```

Comment:
A title tag `h3` would be better here

Reply:
I am curious to know why

Reply:
I meant instead of the <b>, this tag shouldn't be used for styling purposes and we want the title to be in the title tag so it can be identified by screenreaders.